src/main/java/spring/com/service/impl/UserMapperServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package spring.com.service.impl;
+
+import org.springframework.stereotype.Service;
+import spring.com.dto.UserResponseDto;
+import spring.com.model.User;
+
+@Service
+public class UserMapperServiceImpl {
+    public static UserResponseDto dtoBoxing(User user) {
+        UserResponseDto userResponseDto = new UserResponseDto();
+        userResponseDto.setName(user.getName());
+        userResponseDto.setSurname(user.getSurname());
+        userResponseDto.setEmail(user.getEmail());
+        return userResponseDto;
+    }
+
+    public static User dtoUnboxing(UserResponseDto userResponseDto) {
```

Comment:
where do we use this method? Let's remove it)

Reply:
done